test/site_selection.jl

```diff
@@ -28,3 +31,23 @@ end
     @test all([in(sid, [2, 3]) for sid in prefseedsites[prefseedsites.>0]])
     @test all([in(sid, [2, 3]) for sid in prefshadesites[prefshadesites.>0]])
 end
+
+@testset "Guided site selection without ADRIA ecological model" begin
+    criteria_df = ADRIA.sample(dom, 5)  # get scenario dataframe
```

Comment:
I'm not sure how this can run/pass without `dom` being created/assigned to

Reply:
I may have had it already in the environment, I'll add it now

Reply:
Sounds like your running the code directly rather than through the testing infrastructure?

Reply:
I was when I was making sure it was working but I've run the tests using include("./test/site_selection.jl") etc now